batcache-manager.php

```diff
@@ -157,6 +159,11 @@ public function action_update_user( $user_id ) {
 		$this->clear_urls();
 	}
 
+	public function action_update_widget( $instance ) {
+		batcache_flush_all();
```

Comment:
Do `function exists` here, just incase they don't put the batcache stats file in the right place.